### PR TITLE
Feat/implement api endpoint to get listing

### PR DESCRIPTION
**Pull request by dmukherjee**

### CODE REVIEW

File: server/index.js [hunk 1]
```diff
@@ -1,8 +1,17 @@
 const express = require('express');
 
+const Listing = require('../data/mockedDataSearch');
+
 const app = express();
 const PORT = 3001;
 
+app.get('/api/listing/:location', (req, res) => {
+  const queryString = req.url.split('/')[req.url.split('/').length - 1].toLowerCase();
```
Comment (jbgomez) [inline server/index.js]: This can be more easily written as:

`const queryString = req.url.split(':')[1].toLowerCase();`

Comment (jbgomez) [reply server/index.js]: Actually, I think I'm wrong about this. But there's middleware that'll extract that data, I think. Feel free to ignore my comments lol

Comment (jbgomez) [reply server/index.js]: This link says that the location param would look like this:

`req.params.location`

https://stackoverflow.com/a/20089634

**Comment by dmukherjee** [comment]
closes #1 and #2

**Pull request merged**